seed/ts-sdk/circular-references/src/api/resources/ast/types/JsonLike.ts

```diff
@@ -1,5 +1,7 @@
 // This file was auto-generated by Fern from our API Definition.
 
-import type * as SeedApi from "../../../index.js";
+export interface JsonLikeArray extends Array<JsonLike> {}
```

Comment:
Why did we move arrays to be an interface as well instead of just records? Is there a benefit or drawback?